HoloPipelines/pipelines/components/compHttpRequest.py

```diff
@@ -0,0 +1,23 @@
+import requests
+import pathlib
+import sys
+
+
+def sendFilePostRequest(url, inputFile, outputFile):
+    inputFile = str(pathlib.Path(inputFile))
+    outputFile = str(pathlib.Path(outputFile))
+    with open(inputFile, "rb") as inputFileData:
+        file = {"file": inputFileData}
+        try:
+            response = requests.post(url, files=file, timeout=10)
```

Comment:
Add validation for the response status code

Reply:
add check for code 200